WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -0,0 +1,78 @@
+<% provide(:title, "Registrations for #{@competition.name}") %>
+
+<%= form_for @competition, url: competition_registrations_update_all_path(@competition) do |f| %>
+  <table class="table registrations-table selectable-rows">
+    <thead>
+      <tr>
+        <th>WCA id</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Birthday</th>
+        <th>Guests</th>
+        <th>Comments</th>
+        <% @competition.events.each do |event| %>
+          <th>
+            <span class="cubing-icon icon-<%= event.id %>" data-toggle="tooltip" data-placement="top" title="<%= event.name %>"></span>
+          </th>
+        <% end %>
+      </tr>
+    </thead>
+    <tbody>
+      <% @competition.registrations.order(status: :desc).each do |registration| %>
```

Comment:
Registrations are neatly sorted by status, so "accepted" and "pending/rejected/..."; and demarcated as such. But just by looking at the page it isn't clear what is going on, so some sort of tag and/or styling might help here.